python/lsst/ts/standardscripts/maintel/take_triplet_comcam.py

```diff
@@ -0,0 +1,244 @@
+        # plus estimation on processing the data (10s)
+
+        metadata.duration = (
+            self.n_iterations
+            * 3
+            * (
+                self.exposure_time
+                + self.camera.read_out_time
+                + self.camera.shutter_time
+            )
+        )
+        metadata.filter = f"{self.filter}"
+
+    async def configure_camera(self) -> None:
+        """Handle creating ComCam object and waiting for remote to start."""
+        if self._comcam is None:
+            self.log.debug("Creating Camera.")
+
+            self._comcam = ComCam(
+                self.domain, log=self.log, intended_usage=ComCamUsages.TakeImage
+            )
+            await self._comcam.start_task
+        else:
+            self.log.debug("Camera already defined, skipping.")
+
+    async def configure_tcs(self) -> None:
+        """Handle creating MTCS object and waiting for remote to start."""
+        if self.mtcs is None:
+            self.log.debug("Creating MTCS.")
+            self.mtcs = MTCS(
+                domain=self.domain,
+                log=self.log,
+            )
+            await self.mtcs.start_task
+        else:
+            self.log.debug("MTCS already defined, skipping.")
+
+    async def take_triplet(
+        self,
+    ) -> None:
+        """Take triplet (intra focal, extra focal and in-focus)
+        image sequence.
+        """
+        # Take intra focal image
+        self.log.debug("Moving to intra-focal position")
+
+        await self.mtcs.move_camera_hexapod(x=0, y=0, z=self.dz, u=0, v=0)
+
+        self.log.info("Taking intra-focal image")
+
+        await self.camera.take_cwfs(
+            exptime=self.exposure_time,
+            n=1,
+            group_id=self.group_id,
+            filter=self.filter,
+            reason="INTRA" + ("" if self.reason is None else f"_{self.reason}"),
+            program=self.program,
+        )
+
+        self.log.debug("Moving to extra-focal position")
+
+        # Hexapod offsets are relative, so need to move 2x the offset
+        # to get from the intra- to the extra-focal position.
+        z_offset = -(self.dz * 2.0)
+        await self.mtcs.move_camera_hexapod(x=0, y=0, z=z_offset, u=0, v=0)
+
+        self.log.info("Taking extra-focal image")
+
+        # Take extra-focal iamge
+        await self.camera.take_cwfs(
+            exptime=self.exposure_time,
+            n=1,
+            group_id=self.group_id,
+            filter=self.filter,
+            reason="EXTRA" + ("" if self.reason is None else f"_{self.reason}"),
+            program=self.program,
+        )
+
+        self.log.debug("Moving to in-focus position")
+
+        # Move the hexapod back to in focus position
+        await self.mtcs.move_camera_hexapod(x=0, y=0, z=self.dz, u=0, v=0)
+
+        self.log.info("Taking in-focus image")
+
+        # Take in-focus iamge
+        await self.camera.take_cwfs(
```

Comment:
in-focus image should be `take_acq`... The `take_cwfs` is reserved for the intra/extra images.